rotkehlchen/api/v1/encoding.py

```diff
@@ -698,6 +698,14 @@ class ModifiableSettingsSchema(Schema):
     main_currency = FiatAssetField(missing=None)
     # TODO: Add some validation to this field
     date_display_format = fields.String(missing=None)
+    amount_display_format = fields.String(
+        strict=True,
+        validate=webargs.validate.Regexp(
+            regex=r".*%T.*%U.*%D.*",
```

Comment:
So the currency format placeholder is optional?

Reply:
I guess it could be optional. But I may be wrong, do we want it mandatory?

Reply:
no it's fine I guess. Defaults to being at the end I suppose.